docs/pages/upgrading/client-tools-managed-updates.mdx

```diff
@@ -133,6 +133,9 @@ If there is no active session, but a versioned binary exists in the tools direct
 
 ### Cluster-wide using `tctl`
 
+To configure client tool managed updates settings in your cluster, you must have access to the
```

Comment:
This isn't exactly right. `autoupdate_version` is not writable on Cloud, where users can only interact with `autoupdate_config`.